### ISSUE TITLE
Use different key binding for Show All Errors on OS X

### ISSUE THREAD

**Issue by awilke** [niosus/EasyClangComplete#434]
The default shortcut for the "Show All Errors" command is Ctrl+Shift+E. This overrides a helpful global emacs-style shortcut on OS X, which highlights the text from the insertion point to the end of line.

I've tried to "unbind" the shortcut by adding`{ "keys": ["ctrl+shift+e"]}` to my user keymap file. But it seems that Sublime Text doesn't actually release the binding back to the OS when I do this; it just causes the shortcut to not invoke "Show All Errors" anymore.

It would be great if a different binding could be used, since my present workaround is to remove that binding from EasyClangComplete's `Default.sublime-keymap`.

**Comment by niosus** [comment]
Oh, sorry for that. That is what I am afraid of when introducing any shortcuts. Did you try to replace it with something else instead of just trying to disable it? Would the <kbd>cmd</kbd>+<kbd>shift</kbd>+<kbd>e</kbd> work for you?

**Comment by awilke** [comment]
Thanks for the quick response!

Yeah, I tried to set a new shortcut for `ecc_show_all_errors` in my settings, but it just adds that as yet another shortcut which triggers that command.

It looks like Cmd+Shift+E is also taken, as is Ctrl+Cmd+Shift+E. But Ctrl+Cmd+Shift+A is unused, maybe that would work (despite its length)?

**Comment by awilke** [comment]
Agreed, 4 keys is definitely not great. It also looks like Ctrl+Cmd+S is unused, at least with my setup.

**Comment by boazsegev** [comment]
As a side-note, I would also consider automatically invoking the `ecc_show_all_errors` after a file save (if there are any errors).

This will optimize the workflow, so there's no need to use the shortcut if no errors were reported.

**Comment by niosus** [comment]
@boazsegev it is indeed a side note. My initial feeling about this is that it will cause more pain than actual help. We are talking about one keypress versus potentially showing some random errors in std lib upon saving. My current feeling is that there are still many inconsistencies with STL to have that enabled. Is pressing a shortcut that critical? One anyway only needs to do this in case there are errors in the file in the first place. Anyway, feel free to open a different issue and discuss it there if you'd like.

**Comment by boazsegev** [comment]
@niosus ,

Thank you for taking the time to consider my proposal.

> My current feeling is that there are still many inconsistencies with STL to have that enabled...

I don't code in C++ very often, so I guess I never thought about the issue of false positives.

> Is pressing a shortcut that critical? One anyway only needs to do this in case there are errors in the file in the first place...

It isn't as critical as it's either ignored (I forget) or performed together with the `cmd`+`s` shortcut. 

The errors don't pop up like they do on some platforms (such as Xcode), so I sometimes don't notice the errors until I attempt a compilation.

> We are talking about one keypress versus...

This is actually why I made my suggestion. I wish I was able to get my keyboard sequence to be a single shortcut (`cmd`+`s` and than `ctrl`+`shift`+`e` will be condensed into a single `cmd`+`s`).

> feel free to open a different issue and discuss it there if you'd like

I really love your work and don't wish to impose further.

Thank you for your time and effort.
